addOns/ascanrulesBeta/CHANGELOG.md

```diff
@@ -37,6 +37,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
     - The URI field will be in encoded form.
   - XPath Injection
     - Added evidence
+  - Hidden File
+    - Some content checking has been added for .svn/entries as well as detection for wc.db
```

Comment:
It seems like this PR was started from a pretty old base. You might want to revert this, fetch upstream, rebase then re-add the change note in the current "Unreleased" section. (Note: Changes aren't usually two lines, the case you added to here was more of a one off ... have  a look at the entries in the up-to-date file and you'll see what I mean)